README.md

```diff
@@ -49,6 +43,32 @@ Then do following to setup debugging in Chrome:
 - Open Chrome and type following in the address bar: chrome://inspect
 - Click on "Open dedicated DevTools for Node"
 
+### Jest Tips
+
+These tips are from the [Jest CLI docs](https://jestjs.io/docs/en/22.x/cli#running-from-the-command-line)
+
+Run only tests related to changed files based on git:
+
+```shell
+yarn test -o
+```
+
+Find tests related to `fileA.js`:
+
+```shell
+jest --findRelatedTests path/to/fileA.js
```

Comment:
i think this is also `yarn test`

Reply:
really? I have to admit I don't use `--findRelatedTests` tests much but do `-t` or `yarn test -t REGEX` which does similar stuff